prefect_github/mutations.py

```diff
@@ -1,5 +1,5 @@
 """
-This is a module for interacting with GitHub Mutation  tasks.
+This is a module for interacting with GitHub  tasks.
```

Comment:
wow, such nit, so small
```suggestion
This is a module for interacting with GitHub tasks.
```